File: api/ComidaAPI_base/Models/Sugerencia.cs
namespace ComidaAPI.Models;

public class ConjuntoSugerencia
{
    public Alimento? Alimento { get; set; }
    public Bebida? Bebida { get; set; }
    public float CostoTotal { get; set; }
}

public class Sugerencia
{
    public List<ConjuntoSugerencia> Conjuntos { get; set; } = [];
    public float PresupuestoRestante { get; set; }
}

File: app/src/main/java/com/example/comida/persistencia/modelos/Sugerencia.kt
package com.example.comida.persistencia.modelos

data class ConjuntoSugerencia(
    val alimento: Alimento? = null,
    val bebida: Bebida? = null,
    val costoTotal: Float = 0f
)

data class Sugerencia(
    val conjuntos: List<ConjuntoSugerencia> = emptyList(),
    val alimentos: List<Alimento> = emptyList(),
    val bebidas: List<Bebida> = emptyList(),
    val costoTotal: Float = 0f,
    val presupuestoRestante: Float = 0f
) {
    fun dentroDePrresupuesto(presupuesto: Float) = costoTotal <= presupuesto
}